### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by jim-minter**
```release-note
NONE
```
CI secrets have been updated

### CODE REVIEW

File: pkg/fakerp/config.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func GetPluginTemplate() (*pluginapi.Config, error) {
 	if err != nil {
 		return nil, err
 	}
-	genevaPullSecret, err := ioutil.ReadFile("secrets/acr-docker-pull-secert")
+	genevaPullSecret, err := ioutil.ReadFile("secrets/acr-docker-pull-secret")
```
Comment (ehashman) [inline pkg/fakerp/config.go]: Why did this not break anything?

Comment (jim-minter) [reply pkg/fakerp/config.go]: file was misnamed in the secret itself - have duplicated the file to a new name, once this merges and goes back all the branches eventually we'll get rid of the old named file

**Pull request merged**